README.md

```diff
@@ -1,12 +1,123 @@
 # SmartKickers
 
-Definition of Done
-
-- Working code
-- Created code need to be merged
-- Tests implemented
-- Tests passed
-- Code review by devs
-- Sanity check with mentors
-- Acceptance criteria fulfilled 
-- Work presented and accepted
+## Overview
+
+SmartKickers is a project to make your kickers more immersive. It uses Ximea camera to track the ball. 
+
+The following repository contains: 
+* Golang server for processing the data.
+* React client to display game information.
+
+You will also need SmartKickersAI provided by the developers per request. It contains:
+* Ball tracking software.
+* Node server responsible for sending data to the Golang server.
+
+Link to SmartKickerAI
+https://github.com/HackYourCareer/SmartKickersAI.
+
+## Features
+
+* Keeping track of the score.
+
+## Prerequirements
+
+* [python 3.10.5](https://www.python.org/downloads/macos/)
+* [node 16.16.0](https://nodejs.org/en/download/)
+* npm 8.11.0
```

Comment:
You can add information that this version is the minimum supported one like we do [here](https://github.com/kyma-project/busola#prerequisites). My node's version is `v16.14.2` and the code works so that information can be misleading.